core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -344,8 +359,14 @@ var StoryEditorPage = function() {
       nodeDescriptionInputField,
       'NodeDescriptionInputField takes too long to be visible'
     );
-    await nodeDescriptionInputField.clear();
-    await nodeDescriptionInputField.sendKeys(nodeDescription);
+    await action.clear(
+      'Node Description Field', nodeDescriptionInputField
```

Comment:
Can you name this `Node Description Input Field` to be consistent with `action.sendKeys` below?